skrub/_tabular_learner.py

```diff
@@ -129,34 +136,34 @@ def tabular_learner(estimator, n_jobs=None):
                     ('simpleimputer', SimpleImputer(add_indicator=True)),
                     ('logisticregression', LogisticRegression())])
 
-    By applying only the first pipeline step we can see the transformed data
-    that is sent to the supervised estimator (see the :obj:`TableVectorizer`
-    documentation for details):
+    By applying only the first pipeline step we can see the transformed data that is
+    sent to the supervised estimator (see the :obj:`TableVectorizer` documentation for
+    details):
 
-    >>> model.named_steps['tablevectorizer'].transform(X)            # doctest: +SKIP
+    >>> model.named_steps['tablevectorizer'].transform(X)
```

Comment:
I would avoid not testing the docstring. I think this is useful to find out that this is outdated.

Reply:
OK now I understand why we were skipping :)